### PR TITLE
add doro67:regular_fixspc 

### PR DESCRIPTION
**Pull request by diggerdu**
add doro67:regular_fixspc, fix the `SPACE` key disabled issue of doro67:regular

## Description
add doro67:regular_fixspc, fix the `SPACE` key disabled issue of doro67:regular by correct the wrong definition of keymap

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/doro67/regular_fixspc/config.h [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#pragma once
```
Comment (drashna) [inline keyboards/doro67/regular_fixspc/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/doro67/regular_fixspc/config.h (hunk 1 above)
Comment (diggerdu) [reply keyboards/doro67/regular_fixspc/config.h:4]: I have added the header.

File: keyboards/doro67/regular_fixspc/readme.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+This is not the PCB with RGB support. Do not flash RGB firmware for this board. 
+
+**REGULAR PCB (AVAILABILITY: CHINA GB ONLY)*
+If you are a non-china buyer, you probably have the multi PCB or rgb PCB. 
+Please look at the "multi" and "rgb" readme files.
+
+Keyboard Maintainer: [MechMerlin](https://github.com/mechmerlin)  
+Hardware Supported: Doro67 Regular PCB  
+Hardware Availability: [Geekhack GB](https://geekhack.org/index.php?topic=97265.0)
```
Comment (fauxpark) [inline keyboards/doro67/regular_fixspc/readme.md:13]:
```suggestion
* Keyboard Maintainer: [MechMerlin](https://github.com/mechmerlin)
* Hardware Supported: Doro67 Regular PCB
* Hardware Availability: [Geekhack GB](https://geekhack.org/index.php?topic=97265.0)
```

File: keyboards/doro67/regular_fixspc/readme.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+Make example for this keyboard (after setting up your build environment):
+
+    make doro67/regular_fixspc:default
+
+For Via Support:
+
+    make doro67/regular_fixspc:via
+
```
Comment (fauxpark) [inline keyboards/doro67/regular_fixspc/readme.md:22]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by diggerdu** [comment]
@fauxpark Hi the changes have been commited

File: keyboards/doro67/regular_fixspc/readme.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Doro67 Regular PCB 
+
+65% custom keyboard made by 80ultraman/Alf/Backprop Studios available during the China Group Buy. The regular_fixspc version fix the `SPACE` disabled issue in regular version.
+
+This is not the PCB with RGB support. Do not flash RGB firmware for this board. 
+
+**REGULAR PCB (AVAILABILITY: CHINA GB ONLY)*
```
Comment (fauxpark) [inline keyboards/doro67/regular_fixspc/readme.md:7]:
```suggestion
**REGULAR PCB (AVAILABILITY: CHINA GB ONLY)**
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by noroadsleft — COMMENTED** [review]
I don't see what issue this is supposed to solve. I compared the codebase of this addition to the existing `doro67/regular` codebase, and the compilation results are identical.

```
$ shasum doro67_regular*_default.hex doro67_regular*_via.hex
a03758ab1727778773fe0284990ad9a2caad2fa6 *doro67_regular_default.hex
a03758ab1727778773fe0284990ad9a2caad2fa6 *doro67_regular_fixspc_default.hex
c6b7eaa4bf8241d9723881f5a1ee458538f893fc *doro67_regular_fixspc_via.hex
c6b7eaa4bf8241d9723881f5a1ee458538f893fc *doro67_regular_via.hex
```

**Pull request closed without merging**